server/analysis/tests/test_nn.py

```diff
@@ -29,10 +29,10 @@ def setUpClass(cls):
 
     def test_nn_ypreds(self):
         ypreds_round = ['%.3f' % x[0] for x in self.nn_result]
-        expected_ypreds = ['20.503', '22.158', '22.158', '25.692', '24.536', '23.637']
```

Comment:
Could you please help me set both python seed and numpy seed before setting tensorflow seed (line 23) ? 
It should be something like this : 

        random.seed(0)
        np.random.seed(0)
        set_random_seed(0)

Then update the expected results accordingly. 
It should solve the dependency issue between our tests. 

Thanks.

Reply:
No problem. FYI, there is a way to suggest changes. [https://help.github.com/en/articles/incorporating-feedback-in-your-pull-request](url)

Reply:
Thanks, Dongsheng.